.github/workflows/java.yml

```diff
@@ -336,8 +360,17 @@ jobs:
               working-directory: java
               env:
                   GLIDE_CONTAINER_BUILD: true
-                  JAVA_HOME: /usr/lib/jvm/java-${{ matrix.java }}-amazon-corretto.x86_64
-              run: ./gradlew --build-cache --continue build -x javadoc
+              run: |
+                  if [[ "${{ matrix.host.OS }}" == "amazon-linux" ]]; then
+                      export JAVA_HOME=/usr/lib/jvm/java-${{matrix.java}}-amazon-corretto.x86_64
+                  else
+                      # TODO: Fix java version. Java matrix tests are currently broken for all OS and Java 11 is always used.
+                      # On other OS, build auto-downloads Java 11 but this doesn't work on Alpine
+                      # export JAVA_HOME=/usr/lib/jvm/java-${{matrix.java}}-openjdk
+                      export JAVA_HOME=/usr/lib/jvm/java-11-openjdk
```

Comment:
Version isn't hardcoded

Reply:
The java-11-openjdk is a temporary workaround for the issue with java matrix version, as described in the comment.